styles/components/navigation/sideBar.scss

```diff
@@ -5,6 +5,7 @@
   height: calc(100vh - #{$header-height});
   z-index: 0;
   width: 314px;
+  z-index: 2;
```

Comment:
LGTM but:

* Line 115 has .sideNav with z-index:11. Should that be here? Not sure how the two interplay...
* At some point we should get all z-index values into a single framework -- or at least bring them all to the same file.

Reply:
I unified the `z-index` values on c46f93e. I'll keep the `z-index` idea in mind when refactoring the CSS per our convo yesterday!